store/postgres/src/relational/ddl_tests.rs

```diff
@@ -221,8 +221,6 @@ fn generate_ddl() {
     let il = IndexList::mock_thing_index_list();
     let layout = test_layout(THING_GQL);
     let sql = layout.as_ddl(Some(il)).expect("Failed to generate DDL");
-    println!("SQL: {}", sql);
```

Comment:
was this intentionally removed?

Reply:
yeah. used to be one of my debugging statements...